src/reactapp/src/tailwind.config.js

```diff
@@ -0,0 +1,98 @@
+const colors = require('tailwindcss/colors');
```

Comment:
So now we have 2 tailwind setups again? here and in the view/frontend directory?
I just got rid of this one 😅

Reply:
I know you removed it before. But i need tailwind.config.js at that place. Otherwise, vscode will complain about tailwind autocompletion. I could not find a way to resolve that issue. Vscode requires tailwind.config.js at the root folder in order to get the suggestions and all. Enlight me if I am wrong here. 

Remember we have prettier, eslint and tailwind configuration in the root level. To make everything work together, it is better we keep this file here. Otherwise some of the items wont work

Reply:
ok. it's just not ideal to have 3 locations for the tailwind config (in the theme and twice in checkout)